platform/linux/config.cmake

```diff
@@ -1,5 +1,5 @@
 mason_use(glfw VERSION 3.2.1)
-if (WITH_OSMESA OR IS_CI_BUILD)
+if(IS_CI_BUILD AND NOT WITH_EGL)
```

Comment:
I'm a little confused by this - are we installing Mesa somewhere else for EGL CI builds? It appears that we're still setting `MASON_MESA_SUFFIX` for `WITH_EGL` builds in `CMakeLists.txt`.